sql/_05_plcsql/_01_testspec/_02_declaration/_01_parameter/_05_in_out_mode/answers/09_default_value_param.answer

```diff
@@ -94,7 +94,7 @@ expression
 null     
 
 
- +12345
+' +12345'
```

Comment:
It has been reported as a bug. It would be better to wait for the processing result.
http://jira.cubrid.org/browse/CBRD-25750

Reply:
revert back.